ext/date/tests/date_period_bad_iso_format.phpt

```diff
@@ -0,0 +1,29 @@
+--TEST--
+Test bad ISO date formats passed to DatePeriod constructor
+--FILE--
+<?php
+
+try {
+    $perid = new DatePeriod("R4");
```

Comment:
```suggestion
    $period = new DatePeriod("R4");
```
Or drop the variable :)

Reply:
Ah, I don't even know why I added the variables in the first place... :) (I do know indeed: copy-pasted from date_interval_bad_format_leak.phpt)

Reply:
You copied my typo :P

Reply:
Fixed them during applying the changes in this PR :)